src/components/GridItem/gridItemUtils.ts

```diff
@@ -0,0 +1,28 @@
+import { px } from '../../utils';
+import { GridItemProps } from './GridItem';
+
+export enum GridItemAlign {
+  center = 'center',
+  left = 'left',
+  right = 'right',
+}
+
+export const determineColumnSpanClassName = (
+  breakpoint: 'xs' | 'sm' | 'md' | 'lg',
```

Comment:
Since we are using enum for  `GridItemAlign` I believe it will be consistent to create enum with `'xs' | 'sm' | 'md' | 'lg'`